A-set/199A.Hexadecimals_theorrem.cpp

```diff
@@ -0,0 +1,39 @@
+#include<bits/stdc++.h>
+#include<sstream>
+using namespace std;
```

Comment:
Add an extra blank line before this. Do this for the rest of the 4 solutions.

Reply:
Updated. Please check